scripts/bz.py

```diff
@@ -90,7 +98,7 @@ def configure_vs_tools():
         print('ERR: Unable to find VS dev command-line tools')
         sys.exit(1)
 
-    return '"%s" -host_arch=%s -arch=%s' % (vs_devcmd_file, get_arch(), get_arch())
+    return '"%s" -host_arch=%s -arch=%s' % (vs_devcmd_file, get_arch(), get_target_arch())
```

Comment:
I consider this a bug in the original script. I'm not sure why you define `-host_arch` and `-arch` for Windows, or why this is in this special function / this particular spot in the CLI.

I'd suggest to move this to where the macOS cross-compile is added, so you can also build arm64 builds for Windows (assuming there's a cross compiler and necessary libs in VS, as there is on macOS/Xcode).

Reply:
Hey @JannikGM, Thanks for the suggestions. This part was built based on this [script file (Webview's fork)](https://github.com/neutralinojs/webview/blob/3f9742ac25445e183c9807a1feb2984ee97e3a87/script/build.bat#L43) and I think it's used to configure MSVC. Perhaps we can test this section further when we try to generate ARM64 binaries on Windows?